src/cpp/gettensorcommand.cpp

```diff
@@ -31,8 +31,8 @@
 
 using namespace SmartRedis;
 
-const int num_elements = 2;
-const int num_fields = 6;
+const size_t num_elements = 2;
```

Comment:
Good catch here.  I think that ``type_idx``, ``dims_idx``, and ``blob_idx`` are probably better as ``size_t`` as well since ``operator[]`` accepts ``size_t``.

Reply:
Yep. This was generating a warning about signed/unsigned and/or size mismatches once we turned on -Wall.